Rakefile

```diff
@@ -38,12 +38,12 @@ desc 'Generate all stats'
 task generate_stats: [:environment] do
   require File.expand_path(File.dirname(__FILE__) + '/lib/stats_reporter')
   stats_reporter = StatsReporter.new
-  complete_report = <<-REPORT.strip_heredoc
-Stats compiled on #{Time.now.to_date}
-Storage stats for mounts on #{Socket.gethostname}:
-#{stats_reporter.storage_report_text}
-Workflow stats:
-#{stats_reporter.workflow_report_text}
+  complete_report = <<~REPORT.strip_heredoc
+    Stats compiled on #{Time.now.to_date}
+    Storage stats for mounts on #{Socket.gethostname}:
+    #{stats_reporter.storage_report_text}
+    Workflow stats:
+    #{stats_reporter.workflow_report_text}
```

Comment:
i tested something like this in rails console just to be sure, and the old and new implementations do indeed behave identically (i.e. the leading spaces in the new version don't make it into the resultant string).